pages/design/ui-patterns/empty-states.mdx

```diff
@@ -45,3 +45,8 @@ Illustrations should be used **sparingly** in blankslates - only in cases where
 Voice and tone are important to making sure users are not only engaged, but also informed about features. Empty states should explain features _in addition_ to conveying intention. Below is a good example of conveying intention in the primary text and using the secondary text to inform.
 
 ![code of conduct illustration blankslate](https://user-images.githubusercontent.com/6846673/62806616-394b0280-baa8-11e9-8a5e-45f0c51aaa22.png)
+
+## New User Experiences
+For new user experiences, use illustration blankslates to playfully engage the user and introduce the Octocats as symbols of GitHub. Primary text should similarly convey how the feature connects to GitHub's mission. Secondary text should seek to educate the user, but at a simpler, less-technical level.
```

Comment:
```suggestion
For new user experiences, use illustration blankslates to playfully engage the user and introduce the Octocat as a symbol of GitHub. Primary text should similarly convey how the feature connects to GitHub's mission. Secondary text should seek to educate the user, but at a simpler, less-technical level.
```
> Primary text should similarly convey how the feature connects to GitHub's mission. 

The primary text should be use to to introduce/welcome the user to the feature/platform.